File: python1/prime.py
num1=int(input("enter the number "))

if num1>1:
    for i in range(2,int(num1/2)+1):
        if (num1%i)==0:
            print(num1,"it is not prime")
        break
    else:
        print("it is prime",num1)  
else:
    print("it is mot prime number",num1)          

File: c/prime.c
#include<stdio.h>
int main()
{
int n,i,count=0;
printf("enter the number");
scanf("%d",&n);
for(i=1;i<=n;i++)
{
if(n%i==0)
{
count++;
}
if(count==1)
   printf("number is prime");
else
    printf("number is not prime");
}
}
